lib/backend/helpers.go

```diff
@@ -61,3 +61,31 @@ func ReleaseLock(ctx context.Context, backend Backend, lockName string) error {
 	}
 	return nil
 }
+
+// ResetLockTTL resets the TTL on a given lock.
+func ResetLockTTL(ctx context.Context, backend Backend, lockName string) error {
+	if lockName == "" {
+		return trace.BadParameter("missing parameter lock name")
+	}
+	key := []byte(filepath.Join(locksPrefix, lockName))
+	if _, err := backend.Get(ctx, key); err != nil {
```

Comment:
Are you sure that Get call resets dynamodb entry TTL ?

Reply:
According to https://github.com/gravitational/teleport/blob/eebb5b2510fa4110b6efe03bb03f5a936d20b269/lib/backend/helpers.go#L36 it should do just that.

Reply:
That seems wrong to me.  That snippet may be a holdover from some old behavior.  99% certain that `Get` does not affect TTL.

Reply:
Hmm, alright. Do you know of any way to actually refresh TTL?

Reply:
Oh, we can just issue a PUT.

Reply:
Updated